docs/Glossary.en.md

```diff
@@ -0,0 +1,42 @@
+# Glossary
+
+This document compiles terms used within the engine.  
```

Comment:
```suggestion
This document explains terms used within VOICEVOX ENGINE.  
```
It would be less confusing to use the word "compile" which means "gather" here in programming context.
Also, here "engine"（エンジン）exclusively refers to VOICEVOX ENGINE, so it would be better to be explicit here.